### ISSUE TITLE
index out of bounds panic

### ISSUE THREAD

**Issue by AndreKR** [g1eny0ung/parse_link_header#2]
```rust
let link_header = "<https://example.com/events/?&cursor=0:0:1>; rel=\"previous\"; results=\"false\"; cursor=\"0:0:1\", <https://example.com/events/?&cursor=0:100:0>; rel=\"next\"; results=\"true\"; cursor=\"0:100:0\"";

parse_link_header::parse(link_header);
```

```text
thread 'main' panicked at 'index out of bounds: the len is 1 but the index is 1', github.com-1ecc6299db9ec823\parse_link_header-0.2.0\src\lib.rs:94:57
```

![image](https://user-images.githubusercontent.com/1188538/107365574-44d41b00-6add-11eb-98e7-c388bff9cc4e.png)

**Comment by g1eny0ung** [comment]
Should remove the `&` when the param is the first.

`?&cursor=0:0:1` => `?cursor=0:0:1`.

**Comment by AndreKR** [comment]
Oh, I didn't expect `parse_link_header` to parse the actual link. Anyway, I don't have control over the link, that's how it's returned by the [Sentry API](https://docs.sentry.io/api/pagination/) (click the "Http" tab above the code example).

**Comment by g1eny0ung** [comment]
Know about it. I'm already uploading a patch version(`v0.2.1`) to resolve this problem. Thanks for pointing this out. 😆

**Comment by mdharm** [comment]
This issue looks like it was resolved by commit 7a0cba9224c624b47e0a1789557228d804021858 -- should this issue be closed?

**Comment by g1eny0ung** [comment]
> This issue looks like it was resolved by commit [7a0cba9](https://github.com/g1eny0ung/parse_link_header/commit/7a0cba9224c624b47e0a1789557228d804021858) -- should this issue be closed?

Yep. Just waiting for some replies :). I will close now.
